RELEASE/scripts/autoscend/combat/auto_combat_default_stage5.ash

```diff
@@ -21,6 +21,10 @@ string auto_combatDefaultStage5(int round, monster enemy, string text)
 	// Path = gelatinous noob
 	retval = auto_combatGelatinousNoobStage5(round, enemy, text);
 	if(retval != "") return retval;
+
+	// Path = g-lover
+	retval = auto_combatGloverStage5(round, enemy, text);
+	if(retval != "") return retval;
 	
```

Comment:
G-Lover can't use saucestorm, but can use Kaesesossesturm, which is equally effective most of the time. To simplify things I just added a new path specific combat script.